src/engraving/libmscore/measure.cpp

```diff
@@ -3160,6 +3160,10 @@ void Measure::stretchMeasure(qreal targetWidth)
 
     std::multimap<qreal, Segment*> springs;
 
+    //Fraction minTicks = computeTicks();
+    Fraction minTicks = minSysTicks(); // The stretching must be done according to the shortest note *of the system*!
+    qreal minNoteSpace = (score()->noteHeadWidth() + score()->styleMM(Sid::minNoteDistance));
```

Comment:
This one is not needed, not used anywhere, leads to a compiler warning (in MSCV)